src/Components/Track/Track.js

```diff
@@ -11,8 +11,11 @@ class Track extends Component {
   renderAction() {
     return (
       // In the Track.js '-' element, add an onClick property with the value set to the this.removeTrack method.
-      //   I think I need to break out the different values so this.addTrack is on the '+' and '-' is on the this.removeTrack
-      <a className="Track-action" onClick={this.addTrack}>
+      // I think I need to break out the different values so this.addTrack is on the '+' and '-' is on the this.removeTrack
+      <a
+        className="Track-action"
+        onClick={this.props.isRemoval ? this.removeTrack : this.addTrack}
```

Comment:
Gotta use a different action based on if this is a removal track (in the playlist) or an add track (in the search results)